physionet-django/console/templates/console/application_list.html

```diff
@@ -0,0 +1,34 @@
+<div id="searchitems">
```

Comment:
"application_list.html" doesn't give much information about the role of the template. It would be good if the name made it clear that this was a list of credentialing applications with known references.

Reply:
I changed the name of the file to known_references_list.html, that good ?

Reply:
that sounds good, thanks!